### PR TITLE
Fix misleading size for directories

### PR DESCRIPTION
**Pull request by kennanseno**
https://github.com/denoland/deno_website2/pull/627

@lucacasonato

### CODE REVIEW

File: components/Registry.tsx [hunk 1]
```diff
@@ -519,9 +519,14 @@ function DirectoryListing(props: {
...
-                          {entry.size !== undefined
-                            ? bytesToSize(entry.size)
-                            : "N/A"}
+                          {
+                            /** TODO(kennanseno): Directory should pull total size inside it.  */
+                            entry.type === "dir"
+                              ? ""
+                              : entry.size !== undefined
+                              ? bytesToSize(entry.size)
+                              : "N/A"
```
Comment (ry) [inline components/Registry.tsx:528]: How about just blank instead of "N/A" ?

Comment (kennanseno) [reply components/Registry.tsx:528]: empty string could just work to be honest but it's probably better to differentiate between a valid directory and a file for that some reason we can't resolve the size.

Comment (ry) [reply components/Registry.tsx:528]: > a file that for some reason we can't resolve the size

when does that happen?

Comment (kennanseno) [reply components/Registry.tsx]: I'm just assuming based on the existing undefined check to entry.size.

Not a big deal tbh and happy to leave it to empty string either way.

**Comment by Swap76** [comment]
@kennanseno Thanks :-)

**Pull request merged**